test/sqlsmith/mzcompose.py

```diff
@@ -246,5 +246,12 @@ def kill_sqlsmith_with_delay() -> None:
             )
             print(f"From {from_time} until {to_time}")
 
-        shortest_query = min([error["query"] for error in errors], key=len)
-        print(f"Query: {shortest_query}")
+        # The error message indicates a panic, if we happen to get multiple
+        # distinct panics we want to have all the responsible queries instead
+        # of just one:
+        if "server closed the connection unexpectedly" in errors[0]["message"]:
+            for i, error in enumerate(errors, start=1):
+                print(f"Query {i}: {error['query']}")
+        else:
+            shortest_query = min([error["query"] for error in errors], key=len)
+            print(f"Query: {shortest_query}")
```

Comment:
Can you explain the else part, please? Is it just that we want to get a single query, don't care which one it is, and therefore pick the shortest?

Reply:
Yes, the hope is that the shortest query is the easiest to understand the root cause of the issue.